CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## 20.41.1 - 2024-04-22
 
+### Payments
```

Comment:
Can you update this entry to be under the unreleased section?

Reply:
Won't this go out for release next week?

Reply:
Yes, not as part of the (already released) 20.41.1

Reply:
Oh right, updated 👍